tests/contrib/test_kubernetes/test_context.py

```diff
@@ -0,0 +1,33 @@
+from mlem.contrib.kubernetes.context import K8sYamlBuildArgs, K8sYamlGenerator
+from tests.contrib.test_kubernetes.conftest import _cut_empty_lines
+
+
+def test_k8s_yaml_build_args_default(k8s_default_manifest):
```

Comment:
probably worth to move fixtures here so it's more visible (and you dont use them anywhere else)

Reply:
done